src/ui/handlers/menu-ui-handler.ts

```diff
@@ -517,6 +511,7 @@ export class MenuUiHandler extends OptionSelectUiHandler {
       options: communityOptions,
       xOffset: this.optionSelectBg.displayWidth,
       yOffset: this.menuMessageBox.displayHeight + 1,
+      onResize: (w) => (this.communityWindowWidth = w),
```

Comment:
```suggestion
      onResize: (w) => {
        this.communityWindowWidth = w;
        },
```
Resolves the [`lint/suspicious/noAssignInExpressions`](https://biomejs.dev/linter/rules/no-assign-in-expressions/) warning.